library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -215,6 +215,20 @@ button[class="disabled"]:hover,
 	text-decoration: none;
 }
 
+input[type="checkbox"].formButtonDisabled{
+	@include border-radius(3px);
+	&:checked{
+		background-image: url("images/check.png") !important;
```

Comment:
I understand, but this pushes it to someone else's plate to deal with. I realize this may seem minor but as you said you needed to do it like this because there is another !important for which probably should not have been allowed to begin with. The next person will now have to deal with both and that doesn't seem fair to me.

I'll let others way in :)